src/main/kotlin/it/pagopa/ecommerce/PagopaEcommerceCdcServiceApplication.kt

```diff
@@ -0,0 +1,10 @@
+package it.pagopa.ecommerce
```

Comment:
```suggestion
package it.pagopa.ecommerce.cdc
```
let use cdc as root package folder